bundles/net.tourbook/src/net/tourbook/data/SerieData.java

```diff
@@ -144,6 +150,31 @@ public String toString() {
 
       final int maxLen = 10;
 
+    //CUSTOM TRACKS
+      String custTrack = "\n";
+      if(customTracksDefinition!=null && !customTracksDefinition.isEmpty()) {
+         custTrack += "...Definition...len=" + Integer.toString(customTracksDefinition.size()) + "\n";
+         for (final String i : customTracksDefinition.keySet()) {
+            final CustomTrackDefinition item = customTracksDefinition.get(i);
+            custTrack += "  Id=\"" + i + "\" ,Name=\"" + item.getName() + "\"" + " ,Unit=\"" + item.getUnit() + "\"" + "\n";
+         }
+         custTrack += "-- --\n";
+      }
+      if(customTracks!=null && !customTracks.isEmpty()) {
+         for (final String i : customTracks.keySet()) {
+            custTrack += "  Id:\"" + i + "\"    " + (customTracks.get(i) != null         ? Arrays.toString(Arrays.copyOf(customTracks.get(i),          Math.min(customTracks.get(i).length, maxLen)))        : UI.EMPTY_STRING) + "\n"; //$NON-NLS-1$ //$NON-NLS-2$
+            if(customTracksStat!= null && customTracksStat.get(i)!=null) {
+               final CustomTrackStatEntry valE = customTracksStat.get(i);
+               custTrack += "  Id:\"" + i + "\"    " + "[Avg=" + String.format("%.2f", valE.value_Avg);
+               custTrack +=  ";Min=" + String.format("%.2f", valE.value_Min);
+               custTrack +=  ";Max=" + String.format("%.2f", valE.value_Max);
+               custTrack += "]\n";
+            }
+         }
+         custTrack += "  --END CUSTOM TRACKS--  \n";
```

Comment:
@gmngueko
In case you don't know, you can externalize string in Eclipse by going to :
![image](https://user-images.githubusercontent.com/25235665/103788671-57e65d80-4ffc-11eb-9dc1-e069af279d0b.png)